deploy/olm-catalog/nexus-operator/manifests/nexus-operator.clusterserviceversion.yaml

```diff
@@ -283,13 +251,45 @@ spec:
           verbs:
           - get
         serviceAccountName: nexus-operator
+      deployments:
+      - name: nexus-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: nexus-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: nexus-operator
+            spec:
+              containers:
+              - command:
+                - nexus-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: nexus-operator
+                image: quay.io/m88i/nexus-operator:0.4.0
+                imagePullPolicy: Always
+                name: nexus-operator
+                resources: {}
+              serviceAccountName: nexus-operator
     strategy: deployment
   installModes:
   - supported: true
     type: OwnNamespace
   - supported: true
     type: SingleNamespace
-  - supported: false
+  - supported: true
     type: MultiNamespace
```

Comment:
On OLM they set this variable:

```
- name: WATCH_NAMESPACE
                  valueFrom:
                    fieldRef:
                      fieldPath: metadata.annotations['olm.targetNamespaces']
```

It's from the CSV file. So we are good.

Reply:
I'm under the impression we ought to focus on the [OperatorGroup docs](https://docs.openshift.com/container-platform/4.2/operators/understanding_olm/olm-understanding-operatorgroups.html), it seems that they are the actual resources that control this scope thing. I'm not sure if changes to this warrants changes in code as well.

Again, I couldn't test it yet, but I'm under the impression our current `main` functions just watches all namespaces no matter what, but there is a lot of black magic going on with this. Maybe RBAC prunes the access depending on how OLM is configured and the target namespaces. Then even though we think we're watching all namespaces, we're really only watching the ones injected by OLM. Conjecturing, of course.

/me got some reading to do

Reply:
I will keep everything as "true". It will work nevertheless, plus we have backward compatibility.

Setting namespace to "", will always watches everything.